.travis.yml

```diff
@@ -3,12 +3,15 @@ language: ruby
 bundler_args: --without development system_tests
 before_install: rm Gemfile.lock || true
 use_sudo: false
+script: bundle exec rake test
+
 rvm:
   - 1.8.7
   - 1.9.3
   - 2.0.0
   - 2.1.0
-script: bundle exec rake test
+  - 2.2.0
+  - 2.3.0
 env:
```

Comment:
This has turned into an *epic* travis.yml.

I personally don't mind too much, but dang.